integration/bin/tachyon-yarn.sh

```diff
@@ -36,5 +36,7 @@ JAR_HDFS=${PATH_HDFS}/tachyon-assemblies-${VERSION}-jar-with-dependencies.jar
 
 ${HADOOP_HOME}/bin/hadoop fs -put -f ${JAR_LOCAL} ${JAR_HDFS}
 
+export YARN_OPTS="${YARN_OPTS:-${TACHYON_JAVA_OPTS}}"
```

Comment:
It determines which JVM options `tachyon.yarn.Client` is run with - I've added a comment